machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -62,10 +65,11 @@ query search {
 `
 
 var (
-	flgMonitoringEndpoints = pflag.Bool("monitoring-endpoints", false, "generate Endpoints for monitoring")
-	flgBMCConfigMap        = pflag.Bool("bmc-configmap", false, "generate ConfigMap for BMC reverse proxy")
-	flgNodeExporterPort    = pflag.Int32("node-exporter-port", defaultNodeExporterPort, "node-exporter port")
-	flgEtcdMetricsPort     = pflag.Int32("etcd-metrics-port", defaultEtcdMetricsPort, "etcd metrics port")
+	flgMonitoringEndpoints      = pflag.Bool("monitoring-endpoints", false, "generate Endpoints for monitoring")
+	flgBMCReverseProxyConfigMap = pflag.Bool("bmc-configmap", false, "generate ConfigMap for BMC reverse proxy")
+	flgBMCLogCollectorConfigMap = pflag.Bool("log-collector", false, "generate ConfigMap for BMC log collector")
```

Comment:
Sorry, I can not change the existing code. This change due to avoid occure issue running system.
I named the configmap around with a implicit nameing rule.

Reply:
Changed arrong with your request.